src/styles/blocks/catalog.scss

```
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: $smoothTime;
+
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      color: #34568b;
```

don't remember to create variables for colors

```suggestion
      color: $c-color;
```